Updating Google Cloud GS tutorial to align with AWS tutorial (#4791)
pulumi/docs: content/docs/get-started/gcp/begin.md

title: Before You Begin | GCP
h1: Before You Begin
linktitle: Before You Begin

> meta_desc: This page provides an overview on how to get started with Pulumi when starting an Google Cloud project.

Reviewer: ```suggestion
meta_desc: This page provides an overview on how to get started with Pulumi when starting a Google Cloud project.
```

Also, minor, but "get started .. when starting" reads a bit oddly.  Maybe just"Get started with Pulumi and Google Cloud"?

{{< install-dotnet >}}
{{% /choosable %}}

Finally, configure Pulumi with Google Cloud.

### Configure Pulumi to access your Google Cloud account

> Pulumi requires cloud credentials to manage and provision resources. You must use an IAM user or service account that has **Programmatic access** with rights to deploy and manage resources handled through Pulumi. 

Reviewer: ```suggestion
Pulumi requires cloud credentials to manage and provision resources. You must use an IAM user or service account that has **Programmatic access** with rights to deploy and manage resources. 
```

... only a suggestion, since "with rights to deploy ... resources handled through Pulumi" sounds a bit like the account needs rights to do something Pulumi-specific, which isn't the case. (It just needs rights to deploy and manage GCP resources.)  

Also, would it be worth noting that this is specifically a Google Cloud IAM user or service account, and maybe link to a relevant doc?

Author: Incorporated the change in the first suggestion, but I'm not sure we need to link to the Google Cloud IAM docs or add the qualifier than this is a Google Cloud IAM user (as opposed to AWS) as Google uses that acronym as well.

In this guide, you will need an IAM user account with permissions that can create and populate a Cloud Storage bucket, such as those in the predefined Storage Admin (`roles/storage.admin`) or the Storage Legacy Bucket Owner (`roles/storage.legacyBucketOwner`) roles.

> When developing locally, we recommend that you install the [Google Cloud SDK](https://cloud.google.com/sdk/install) and login with the `gcloud` CLI tool as shown below.

Reviewer: ```suggestion
When developing locally, we recommend that you install the [Google Cloud SDK](https://cloud.google.com/sdk/install) and log in with the `gcloud` CLI tool as shown below.
```
